.travis.yml

```diff
@@ -1,10 +1,21 @@
 language: cpp
 compiler:
   - gcc
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: LEGACY=false
+    - os: linux
+      env: LEGACY=true
+	
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libboost-all-dev
   - sudo apt-get install maven
+  - sudo apt-get install libgtest-dev google-mock
   - sudo pip install cpp-coveralls wheel
```

Comment:
Will this only be for ubnutu 16.04 and above, or does this work for linux 14.04 also?

Reply:
it works on 14.04 too, but the packages are broken